### PR TITLE
Azure sentinel error handler bug

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/46083

## Description
This PR fixes a bug in the error handler function in the azure sentinel integration

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by GuyAfik** [comment]
failed unit-test - https://app.circleci.com/pipelines/github/demisto/content/141256/workflows/246526e9-84f1-469d-833c-4d7ae4ef1a47/jobs/475002

File: Packs/AzureSentinel/ReleaseNotes/1_3_1.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Azure Sentinel
+- Fixed an issue that when trying to decode a 'None' type response content in error_handler, then a JSONDecodeError is raised.
```
Comment (ShahafBenYakir) [inline Packs/AzureSentinel/ReleaseNotes/1_3_1.md:4]:
```suggestion
- Fixed an issue where the integration failed with 'JSONDecodeError' in cases of a 'None' type response.
```

File: Packs/AzureSentinel/Integrations/AzureSentinel/Pipfile [hunk 2]
```diff
@@ -4,17 +4,35 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-pylint = "*"
-pytest = "==5.0.1"
-pytest-mock = "*"
-requests-mock = "*"
-pytest-asyncio = "*"
-cryptography = "*"
+flake8 = "==3.9.2"
```
Comment (ShahafBenYakir) [inline Packs/AzureSentinel/Integrations/AzureSentinel/Pipfile:7]: Why have we changed this file?

**Review by ShahafBenYakir — COMMENTED** [review]
Very nice fix.

File: Packs/AzureSentinel/Integrations/AzureSentinel/Pipfile (hunk 2 above)
Comment (GuyAfik) [reply Packs/AzureSentinel/Integrations/AzureSentinel/Pipfile:7]: The azure sentinel integration runs with the following docker image - ```demisto/crypto:1.0.0.24380```

So this change is simply to update the Pipfile to the same Pipfile used in the docker image - https://github.com/demisto/dockerfiles/tree/master/docker/crypto

So other developers can quickly create a virtual environment with the most updated python version and dependancies when updating this integration somewhere in the future

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**